src/main/java/com/descope/sdk/mgmt/TenantService.java

```diff
@@ -59,4 +103,19 @@ void createWithId(String id, String name, List<String> selfProvisioningDomains)
    * @throws DescopeException in case of errors
    */
   List<Tenant> loadAll() throws DescopeException;
+
+  /**
+   * Search all users according to given filters.
+   *
+   * @param request The options optional parameter allows to fine-tune the search
+   *                filters and
+   *                results. Using nil will result in a filter-less query with a
+   *                set amount of results.
+   * @return {@link Tenant Tenant}
```

Comment:
Should return `{@link List} of {@link Tenant}`

Reply:
Done